python/remote_api/src/lynx/kite.py

```diff
@@ -884,6 +884,39 @@ def f(**kwargs):
     _add_documentation_to_operation(f, name)
     return f
 
+  @property
+  def vertices(self) -> 'SingleOutputAtomicBox':
+    '''Returns the vertices as a table.'''
+    return self._select_all('vertices')
+
+  @property
+  def edges(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edges as a table.'''
+    return self._select_all('edges')
+
+  @property
+  def graph_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the graph attributes as a table.'''
+    return self._select_all('graph_attributes')
+
+  @property
+  def edge_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edge attributes as a table.'''
+    return self._select_all('edge_attributes')
+
+  def segmentations(self, name: str) -> 'SingleOutputAtomicBox':
```

Comment:
Sorry, just minor tweaks. I'd do them but I'm not sure if I can commit to your fork.
```suggestion
  def segmentation(self, name: str) -> 'SingleOutputAtomicBox':
```

It returns a single segmentation so let's call it `segmentation()`.

Reply:
Done.